armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -218,31 +257,41 @@ EOF
 # retain journal logs between reboots 
 ln -sf /mnt/ssd/system/journal/ /var/log/journal
 
+# make bbb scripts executable by sudo
```

Comment:
Nit: I think `with sudo` would be more clear than `by sudo`. (`sudo` is not a user, but a tool to execute commands as the super-user.)